test/cpp/jit/test_memory_planning.cpp

```diff
@@ -432,5 +432,32 @@ TEST(MemoryPlannerTest, LSTMNaive) {
   testLSTM(expected_storage, expected_allocs, Strategy::NAIVE);
 }
 
+TEST(MemoryPlannerTest, SmallLinearScan) {
+  StorageAttrs expected_storage = {1344, DeviceType::CPU};
+  std::vector<AllocAttrs> expected_allocs = {
+      {448, 0, TTP(((Vec{10, 10})), ((Vec{10, 1})))},
+      {448, 448, TTP(((Vec{10, 10})), ((Vec{10, 1})))},
+      {448, 896, TTP(((Vec{10, 10})), ((Vec{10, 1})))},
+  };
+  testSmall(expected_storage, expected_allocs, Strategy::LINEAR_SCAN);
```

Comment:
I take it all strategies are tested with only these two?

Reply:
correct. longer models would've inflated the verification spec data too much i think